src/applications/financial-status-report/components/PreSubmitSignature.jsx

```diff
@@ -139,6 +140,19 @@ const PreSubmitSignature = ({
           <li>My bankruptcy history</li>
         </ul>
 
+        <p>
+          I have read and accept the
+          <a
+            target="_blank"
+            rel="noopener noreferrer"
+            className="vads-u-margin-left--0p5"
+            href={`${environment.BASE_URL}/privacy-policy`}
+          >
+            privacy policy
+          </a>
+          .
+        </p>
+
```

Comment:
Isn't this normally a popup? I guess opening it in a new tab is ok, but I think for a11y you need some `sr-only` text to indicate that it's opening in a new tab.

Reply:
For anyone look at this in the future I found this helpful [article](https://codersblock.com/blog/external-links-new-tabs-and-accessibility/) explaining this exact topic. One of the fixes is to add an extra label then hide it with a utility class like `sr-only` like @Mottie mentioned above.

Will implement this first thing in the AM. @joshkimux do we have a standard for content like this?

Reply:
Currently, it's being communicated through an `aria-label`, which makes sense because interactive elements with `sr-only` text can act awkwardly on mobile screen readers.

```html
<a aria-label="Privacy policy, will open in new tab" target="_blank" href="/privacy-policy/">privacy policy</a>
````

That being said, if we have the freedom to do so, I would actually recommend writing this out as "I have read and accept the privacy policy (opens in a new tab)" instead of hiding it to screen reader only users. This helps Veterans who use other assistive technologies (think magnification tools) and Veterans with cognitive/learning disabilities be able to anticipate the change in context.

For example:
```html
<a target="_blank" href="url here">privacy policy (opens in new tab)</a>
```